playground/pages/edit.tsx

```diff
@@ -162,9 +162,20 @@ export const Edit = (props: { horizontal: boolean }) => {
     !!scratchpad() ? 10 : 1000,
   );
 
+  compiler.addEventListener('message', ({ data }) => {
+    const { event, imports, error } = data;
+
+    if (error) return console.error(error);
+
+    if (event === 'IMPORTS') {
+      console.log(imports);
+    }
+  });
```

Comment:
What is this for?

Reply:
oups I believe I must have put it there at the beginning to test it and forgot to remove it